types/node-geocoder/tslint.json

```diff
@@ -1 +1,6 @@
-{ "extends": "@definitelytyped/dtslint/dt.json" }
+{ 
+  "extends": "@definitelytyped/dtslint/dt.json",
+  "rules": {
+    "no-outside-dependencies": false
+  }
```

Comment:
This change was mandatory as the import from `node-fetch` introduced a false positive indicating that `form-data` is not present in the corresponding `package.json`. Adding `form-data` as a dependency directly did not resolve this issue. Similar issues have already been reported earlier (see microsoft/dtslint#315).